_episodes/05-loops-and-functions.md

```diff
@@ -257,11 +257,11 @@ A more elegant solution would be to create a reusable tool that performs this
 task with minimum input from the user. To do this, we are going to turn the code
 we've already written into a function.
 
-Functions are reusable, self-contained pieces of code that are called with a
+Functions are reusable, self-contained pieces of code that can be called with a
```

Comment:
I'm not sure about the need to introduce ambiguity to this sentence by using the uncertain "can be" instead of the definite "are". Please explain.